### DISCUSSION TITLE
Support for custom renderers

### DISCUSSION THREAD

**Discussion by vjr12** [dowjones/react-dropdown-tree-select#402] [Ideas]
Is it possible to somehow display a table like structure. Any custom renderers available? Something like this?

![Screenshot from 2020-07-26 02-30-30 (1)](https://user-images.githubusercontent.com/42905121/88466337-e486de80-cee8-11ea-8760-1924ce2e1c25.png)

**Comment by mrchief** [comment]
@vjr12 this is not possible yet. With the next version that uses hooks, it would be possible but that's lagging on development.

**Comment by vjr12** [comment]
@mrchief Any pointers in implementing the same using the current version? Like how to begin with

**Comment by mrchief** [comment]
You can look at https://github.com/dowjones/react-dropdown-tree-select/issues/337 or https://github.com/dowjones/react-dropdown-tree-select/issues/293 for ideas. 

You can also fork this repo and change the rendering to fit your needs.
